recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ requirements:
     - dcw-gmt  # [not win]
     - ffmpeg
     - graphicsmagick
+    - geos
```

Comment:
Two suggestions:

1. Move `geos` up to lin 38, so that libraries `gdal` and `geos` are together
2. Also add `geos` as a `host` dependency at line24.